app/controllers/admins/users_controller.rb

```diff
@@ -19,6 +19,7 @@ def create
     if @user.save
       success_create_message
       redirect_to admins_users_path
+
     else
       error_message
       render :new
```

Comment:
Empty line not necessary, pain attention on unnecessary modifications. It can mess up the code.